src/rules/max-line-length/README.md

```diff
@@ -74,3 +74,37 @@ a { color: pink; background: orange; }
  */
 a { color: pink; /* this comment is also long but not on its own line */ }
 ```
+
+### `ignore: ["comments"]`
+
+Only enforce the line-length limit for lines that are not commented out.
```

Comment:
@davidtheclark What do you think about this:

If this option is set, this rule will check line-length limit only for lines that are *not* comments and lines that are *not* started with a comment. That is comment lines will be skipped.